### DISCUSSION TITLE
Chia plotter Bottlenck search Ryzen 3600,32Gb,1TB NVME

### DISCUSSION THREAD

**Discussion by VaperDude007** [Chia-Network/chia-blockchain#4779] [Q&A]
Hello Fellow Chia lovers

I am still plotting a small plot for almost 24 hours (no plot ready !!)

I think there is a problem with the read or write!

3 threads per plot
5 GB of mem

These were the results:
Plot 1:
Fase 1 : 11660 seconds (194min)
Fase 2: 6423
Fase 3: 40854 (???)

Plot 2:
Fase 1: 30000
Fase 2: 13544

Plot 3:
Fase 1: 41000 (683min)

I have 12 threads so there is no problem there
Memory i have enoough

Looks like the writing is to slow. The NVME drive reads and writes below 100mb per second.

Almost 24 hours have passed and plot 1 will be ready

Anybody any ideas?

Thanks Folks

**Comment by lvcivs** [comment]
What is the model of the NVME disk?

Some people report really slow plotting with QLC models. You should always choose MLC or TLC. Check this guide: https://chiadecentral.com/chia-blockchain-ssd-buying-guide/
